### PR TITLE
DRUP-239: added listJobFiles method to JobsAPI client

### PR DESCRIPTION
**Pull request by PavelLoparev**

### CODE REVIEW

File: src/Jobs/Params/ListJobsFilesParameters.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Smartling\Jobs\Params;
+
+use Smartling\Parameters\BaseParameters;
+
+class ListJobFilesParameters extends BaseParameters
+{
+    public function __construct($limit = null, $offset = null)
```
Comment (dimitrystd) [inline src/Jobs/Params/ListJobsFilesParameters.php:9]: In java we decided to make `limit` and `offset` as mandatory to force developer to think about pagination. I propose to do the same in new SDK methods.

File: tests/unit/BaseApiAbstractTest.php [hunk 2]
```diff
@@ -28,7 +28,7 @@ public function testNoUserAgentExtensions()
         $this->assertNotFalse(
             \strpos(
                 $http_client->getConfig()['headers']['User-Agent'],
-                'smartling-api-sdk-php/3.9.2 (no extensions) GuzzleHttp/6'
+                'smartling-api-sdk-php/3.9.3 (no extensions) GuzzleHttp/6'
```
Comment (dimitrystd) [inline tests/unit/BaseApiAbstractTest.php:31]: Can you generate the final string using `CLIENT_LIB_ID_VERSION`? Just want to minimize such noisy changes.

**Review by vsolovei-smartling — APPROVED** [review]

File: src/Jobs/Params/ListJobsFilesParameters.php (hunk 1 above)
Comment (PavelLoparev) [reply src/Jobs/Params/ListJobsFilesParameters.php:9]: Fixed

File: tests/unit/BaseApiAbstractTest.php (hunk 2 above)
Comment (PavelLoparev) [reply tests/unit/BaseApiAbstractTest.php:31]: I believe it's fixed in `4.0.0-g7` branch. seems like `3.0.0-g6` is only used by Drupal for now. i don't expect more changes here in the nearest future

**Pull request merged**